llvm/test/Transforms/LoadStoreVectorizer/X86/massive_indirection.ll

```diff
@@ -0,0 +1,63 @@
+; RUN: opt %s -mtriple=x86_64-unknown-linux-gnu -passes=load-store-vectorizer -mcpu=skx -S -o %t.out.ll
+; RUN: FileCheck -input-file=%t.out.ll %s
+
+; This test verifies that the vectorizer can handle an extended sequence of
+; getelementptr instructions and generate longer vectors. With special handling,
+; some elements can still be vectorized even if they require looking up the
+; common underlying object deeper than 6 levels from the original pointer.
+
+; The test below is the simplified version of actual performance oriented
+; workload; the offsets in getelementptr instructins are similar or same for
```

Comment:
```suggestion
; workload; the offsets in getelementptr instructions are similar or same for
```

Reply:
the misprint is fixed: https://github.com/llvm/llvm-project/pull/114501/commits/91a74c80e89ff46dcaf15dad30f9667e082d5979